src/react/hooks/__tests__/useBackgroundQuery.test.tsx

```diff
@@ -2201,22 +2206,169 @@ describe('useBackgroundQuery', () => {
         expect(todo1).toHaveTextContent('Clean room');
       });
     });
-  });
-  describe.skip('type tests', () => {
-    it('disallows returnPartialData in BackgroundQueryHookOptions', () => {
-      const { query } = renderIntegrationTest();
 
-      // @ts-expect-error should not allow returnPartialData in options
-      useBackgroundQuery(query, { returnPartialData: true });
-    });
+    // refetchWritePolicy
+    // it('honors refetchWritePolicy set to "merge"', async () => {
+
+    // });
```

Comment:
```suggestion
    it.todo('honors refetchWritePolicy set to "merge"')
```

I meant to mention this in my previous review, but if you want a really nice DX for tests that you have left todo, you can use [`it.todo`](https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly#testtodo). It will skip the test and show a pencil icon in the test summary.

Reply:
Oh nice! Love that, TIL 🎉